src/test/scala/io/archivesunleashed/app/WriteGEXFTest.scala

```diff
@@ -45,7 +45,7 @@ class WriteGEXFTest extends FunSuite with BeforeAndAfter{
       sc = new SparkContext(conf)
     }
 
-  test("creates the file") {
+  test("Creates the gexf file") {
```

Comment:
If we want to be pedantic we could rename to `GEXF` for consistency with `HTML` etc?